module_auto_update/README.rst

```diff
@@ -63,7 +63,10 @@ the method ``_save_installed_checksums`` can be invoked in a situation
 where one is sure all modules on disk are installed and up-to-date in the
 database.
 
-An easy way to invoke this upgrade mechanism is by issuing the following
+To invoke the upgrade mechanism, navigate to *Apps* menu and use the
+*Auto-Upgrade Modules* button, available only in developer mode.
+
+Another easy way to invoke this upgrade mechanism is by issuing the following
```

Comment:
Please add a note saying that a server restart is highly recommended to take new python code into account.

Reply:
Done